sdk/src/Core/Properties/AssemblyInfo.cs

```diff
@@ -3,7 +3,7 @@
 using System.Runtime.CompilerServices;
 
 [assembly: ComVisible(false)]
-[assembly: CLSCompliant(true)]
+[assembly: CLSCompliant(false)]
```

Comment:
Wondering if you have any ideas on making this code (and other assemblies) CLS compliant instead of setting it to `false`?

Reply:
Usages of Microsoft.AspNetCore.Http.HttpRequest and Microsoft.Extensions.Configuration.IConfiguration are both not CLS compliant.  Given those exposed publicly, I don't believe we have a good path forward to indicate that these assemblies can be made to be CLS compliant.

Reply:
Taking a second look at it, I was able to apply CLSCompliant(false) to methods that were were in violation.  This leaves the assembly CLSCompliant, with small handful of methods and classes that are not.

Reply:
Yup. That's better. Thanks!